sql/plsql/PlSqlLexer.g4

```diff
@@ -2511,7 +2512,7 @@ SPACES: [ \t\r\n]+ -> channel(HIDDEN);
 
 fragment NEWLINE_EOF    : NEWLINE | EOF;
 fragment QUESTION_MARK  : '?';
-fragment SIMPLE_LETTER  : [A-Z];
+fragment SIMPLE_LETTER  : [\p{Alpha}\p{General_Category=Other_Letter}];
```

Comment:
Could you please add a test on identifiers that consist of letters from these caterogies?

Reply:
There is one in the commit (https://github.com/antlr/grammars-v4/pull/4012/commits/7e2337b1f687190d67791dd36cc151889840dd63): sql/plsql/examples/lexer02.sql